### ISSUE TITLE
missing key 'baseVersion'

### ISSUE THREAD

**Issue by danielwossenu** [kiwiz/gkeepapi#18]
I'm getting the following error when running keep.login():

line 851, in load
self._version = raw['baseVersion']
KeyError: 'baseVersion'

The login is successful but when it pulls all the notes and it's setting the properties, it looks like some notes don't have an entry for 'baseVersion' in 'raw' so it fails at the initial sync. Not sure why they wouldn't have a baseVersion or if they do, why it wouldn't get pulled from the server. I looked through the 'raw' dict and it's not every note, and I'm not sure what's different about the ones that don't have it.

**Comment by brunopizzul** [comment]
Same error here,

import gkeepapi
keep = gkeepapi.Keep()
success = keep.login('**********.***', '********')
note = keep.createNote('Todo', 'Eat breakfast')
note.pinned = True
note.color = gkeepapi.node.ColorValue.Red
keep.sync()

File "C:\Python36\lib\site-packages\gkeepapi\node.py", line 851, in load
    self._version = raw['baseVersion']
KeyError: 'baseVersion'

**Comment by kiwiz** [comment]
Hi @brunopizzul @danielwossenu @aFrankLion Could any of you provide an example of a node that has `baseVersion` and does not? Unable to repro atm, but I'm hoping I can find a correlation.

**Comment by aFrankLion** [comment]
@kiwiz, yes, please find below the 'raw' for a list that lacks a baseVersion.

```python
{'kind': 'notes#node', 'id': '16234c92e2c.0011ee21a88764aa', 'serverId': '1wt47IlITu2KN51RCfhvHwKFDinh8HBTIhr0F267F0MqozGCZV8nYZQ_Kwo1nPrASh3mC', 'parentId': 'root', 'type': 'LIST', 'timestamps': {'kind': 'notes#timestamps', 'created': '2018-03-17T16:27:02.948Z', 'updated': '2018-03-17T16:29:19.244Z', 'trashed': '1970-01-01T00:00:00.000Z', 'userEdited': '2018-03-17T16:29:18.601Z'}, 'title': 'Tiles', 'nodeSettings': {'newListItemPlacement': 'BOTTOM', 'checkedListItemsPolicy': 'GRAVEYARD', 'graveyardState': 'COLLAPSED'}, 'isArchived': False, 'isPinned': False, 'color': 'DEFAULT', 'sortValue': '8403939100', 'annotationsGroup': {'kind': 'notes#annotationsGroup'}, 'lastModifierEmail': '*************@gmail.com', 'moved': '1'}
```

**Comment by kiwiz** [comment]
The `'moved': '1'` key looks new. Were any operations performed on this list (how was it created, etc)?

**Comment by danielwossenu** [comment]
@kiwiz here is mine

`{u'kind': u'notes#node', u'annotationsGroup': {u'kind': u'notes#annotationsGroup'}, u'isPinned': False, u'title': u'Test note', u'color': u'DEFAULT', u'nodeSettings': {u'newListItemPlacement': u'BOTTOM', u'graveyardState': u'EXPANDED', u'checkedListItemsPolicy': u'GRAVEYARD'}, u'moved': u'1', u'sortValue': u'279969792', u'lastModifierEmail': u'******@gmail.com', u'parentId': u'root', u'isArchived': False, u'timestamps': {u'userEdited': u'2018-03-17T16:45:46.220Z', u'kind': u'notes#timestamps', u'updated': u'2018-03-17T16:45:47.194Z', u'trashed': u'1970-01-01T00:00:00.000Z', u'created': u'2018-03-17T14:42:37.179Z'}, u'serverId': u'*******', u'type': u'LIST', u'id': u'*********'}`

**Comment by aFrankLion** [comment]
I made the previously supplied list titled 'Tile' with a gkeepapi call. I just made a new list 'Debug List' via the Google Keep website in Chrome and re-ran the keep.login() command, yielding:

```python
{'kind': 'notes#node', 'id': '1521302510166.1302643175', 'serverId': '*************', 'parentId': 'root', 'type': 'LIST', 'timestamps': {'kind': 'notes#timestamps', 'created': '2018-03-17T16:49:09.106Z', 'updated': '2018-03-17T16:49:19.190Z', 'trashed': '1970-01-01T00:00:00.000Z', 'userEdited': '2018-03-17T16:49:18.973Z'}, 'title': 'Debug List', 'nodeSettings': {'newListItemPlacement': 'BOTTOM', 'checkedListItemsPolicy': 'GRAVEYARD', 'graveyardState': 'EXPANDED'}, 'isArchived': False, 'isPinned': False, 'sortValue': '8955362644', 'annotationsGroup': {'kind': 'notes#annotationsGroup'}, 'lastModifierEmail': '*************@gmail.com', 'moved': '1'}
```

**Comment by kiwiz** [comment]
Merged #17 and uploaded 0.10.4 to Pypi. Thanks all!

If anyone's got time to poke around, try to see what results in a Node with `'moved': '1'`.
